core/jazz_test-lambda/test/test.js

```diff
@@ -16,13 +15,186 @@
 // =========================================================================
 
 const assert = require('chai').assert;
+const expect = require('chai').expect;
 const index = require('../index');
+const request = require('request');
+const awsContext = require('aws-lambda-mock-context');
+const sinon = require('sinon');
+const logger = require("../components/logger.js");
+const AWS = require('aws-sdk-mock');
+var context, event
+describe('handler', () => {
+  beforeEach(() => {
+    event = {
+      "method": "POST",
+      "stage": "test",
+      "query": {
+        "service_name": "jazz-service",
+        "username": "xyz",
+        "last_evaluated_key": undefined
+      },
+      "body": {
+        "functionARN": "arn:aws:lambda:us-east-1:192006145812:function:jazz20180621-jazztest-test01-prod:3",
+        "inputJSON": {
+          "name": "applesdadasdasd777"
+        }
+      }
+    }
+    context = {};
+  })
+  it("should throw error if method is not POST", (done) => {
```

Comment:
new line here and before every new it, describe, etc

Reply:
done